blockchain/consensus/deleg_cns/src/composition.rs

```diff
@@ -40,9 +38,9 @@ where
         info!("Starting the delegated consensus proposer...");
         let sm = state_manager.clone();
         let mp = mpool.clone();
-        let mining_tasks = proposer.spawn(sm, mp, submitter).await?;
-        Ok((consensus, mining_tasks))
+        proposer.spawn(sm, mp, submitter, services).await?;
+        Ok(consensus)
     } else {
-        Ok((consensus, vec![]))
+        Ok(consensus)
     }
```

Comment:
This could be simplified, i.e. return `Ok(consensus)` in the end, no `else` needed, unless I'm missing something.

Reply:
Fixed